### DISCUSSION TITLE
Default context menu for TreeItem

### DISCUSSION THREAD

**Discussion by gabrielmaldi** [microsoft/vscode-discussions#997] [Extension Development QnA] [answered]
Is there a way to get the default VS Code context menu (in `view/item/context`) for a `TreeItem`?

<img width="373" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/3728897/9d345d2d-f6e7-410c-b813-e582e8f007e3">

It would be nice if the menu were derived from `resourceUri` just like the icon is.

**Comment by isidorn** [comment, marked as answer]
Not possible right now.
